### ISSUE TITLE
Registering blueprint fails on windows

### ISSUE THREAD

**Issue by onazaruk** [tadams42/flask-marshmallow-openapi#4]
The `flask run` fails with error:
```
  File "...\python3.11\latest\Lib\site-packages\werkzeug\routing\rules.py", line 457, in __init__
    raise ValueError(f"URL rule '{string}' must start with a slash.")
ValueError: URL rule '\v1\docs/static/<path:filename>' must start with a slash.
```

Note how the url is started with "/" (backslash) instead of "\" (slash).

In the `middleware.py` on line 231 `full_url_prefix` is created using `Path()`. I am not sure why `pathlib.Path` is used in url construction there, but on Windows `str(Path("/"))` is resolved to `"\"` (backslash) which later fails in flask.

Can this please be fixed?

**Comment by tadams42** [comment]
You are absolutely correct. Thnx for reporting this.

I believe `Path` was used for convenience of initial implementation and I latter forgot to do this correctly.

Fixed in latest version.

**Comment by onazaruk** [comment]
Thanks for the quick fix, but with the new version (`0.6.2`) the interpreter fails on import:
```
  File "...\flask_marshmallow_openapi\middleware.py", line 312, in OpenAPI
    def collect_static(self, destination_dir: str | Path):
                                                    ^^^^
NameError: name 'Path' is not defined
```

[Type checking doc](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING) says that the first type annotation for module hidden behind the `TYPE_CHECKING` flag must be enclosed in quotes. E.g:
 
```
def collect_static(self, destination_dir: 'str | Path'):
```

Should this be applied in this case as well?

**Comment by tadams42** [comment]
Ugh :man_facepalming: ... let me fix this really quickly...

**Comment by tadams42** [comment]
Done, fixed in new v0.6.3.

thnx for reporting this!
